instrumentation/src/config.c

```diff
@@ -72,10 +77,14 @@ void split_on_eq(char *string, struct kv *pair) {
     pair->v = string;
 }
 
-bool eq_true(char *v) {
+bool str_eq_true(char *v) {
     return v != NULL && !streq("false", v) && !streq("FALSE", v) && !streq("0", v);
 }
 
+bool str_eq_false(char *v) {
```

Comment:
I'm going to consolidate these two functions into `str_to_bool(char *v, bool default)` in a separate PR.